docs/crio.8.md

```diff
@@ -233,7 +233,7 @@ For the bind-mount conditions, only mounts explicitly requested by Kubernetes co
 
 **--grpc-max-send-msg-size**="": maximum grpc receive message size (default: 16 * 1024 * 1024)
 
-**--host-ip**="": host IP considered as the primary IP to use by CRI-O for things such as host network IP (default: "")
+**--host-ip**="": Host IPs are the addresses to be used for the host network and can be specified up to two times (default: [])
```

Comment:
Hm, I thought we would stay backwards compatible with that change, since the only thing that changes is that we can define the flag multiple times. The previous behavior should stay (and that's why I did not rename it as well).

I can for sure add a new option, or I thought to wrap a small struct around the two IPs (4 and 6) and parse it from the CLI.

Reply:
hm I guess because it's a CLI flag it shouldn't be a problem. I didn't realize we didn't have a config var. nevermind I am fine with this